iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/Chmod.java

```diff
@@ -50,6 +50,12 @@ public Chmod(Argument chmodArg, Argument permissionsArg, String permissions) {
     this.permissions = parsePermissions(permissions);
   }
 
+  public Chmod(String permissions) {
+    this.chmodArg = null;
```

Comment:
It's better to call `this(null, null, permissions)` - then the construction logic is one place